addons/mass_mailing/models/mailing.py

```diff
@@ -65,6 +65,8 @@ def default_get(self, fields):
                 mailing_list = self.env['mailing.list'].search([], limit=2)
                 if len(mailing_list) == 1:
                     vals['contact_list_ids'] = [(6, 0, [mailing_list.id])]
+        if 'is_outgoing_server_enabled' in fields:
```

Comment:
Would prefer having a computed field. You will maybe have to set a fake trigger, sometimes computed field without real triggers are not computed before hitting create ( in new mode), to check.

Reply:
Changed to a computed field, appears to be working without having to set a fake trigger.